pyEpiabm/pyEpiabm/household_sweep.py

```diff
@@ -0,0 +1,56 @@
+#
+# Infection due to contact within households
+#
+
+from .abstract_sweep import AbstractSweep
+from .population import Population
+from .covidsim_helpers import CovidsimHelpers
+from .parameters import Parameters
+import random
+
+
+class HouseholdSweep(AbstractSweep):
+    '''Class to run the intra-household infections
+    as part of the sweep function. Takes an individual
+    person as input and tests a infection event against each
+    susceptible member of their household. The resulting
+    exposed person is added to an infection queue.
+
+    : param infected: Person
+    : return
+    '''
+
+    def __call__(self, time: float, population: Population):
+        '''
+        Given a population structure, loops over infected members
+        and considers whether they infected household members based
+        on individual, and spatial infectiousness and susceptibility.
+        '''
+        timestep = int(time * Parameters.instance().time_steps_per_day)
```

Comment:
I don't really know, its something related to the parameters file, better check with Nick

Reply:
@NicholasFan235 can you adivse?

Reply:
Covidsim does it this way (iterates timesteps, converts to time in days, but within many sweeps converts back to timesteps), not sure I like it.

Parameters uses a 'Singleton' pattern so it can be accessed everywhere without being passed around. It's sort of a 'better practice global'